app/js/LocalController.js

```diff
@@ -79,26 +79,22 @@ angular.module('makeithappen').controller('LocalController', ['$scope', function
         "items": $scope.schema
     };
     $scope.usersUiSchema = {
-        "elements": [
+        "type": "Control",
+        "scope": {
+            "$ref": "#"
+        },
+        "columns": [
             {
-                "type": "Control",
```

Comment:
No, columns currently don't have any type (just like it was before this PR). What I've actually changed here it to remove the restriction that the top level element has no type attached, but an elements attribute must be present. Such a top-level entry point does not exist anymore, any UI element can be used now and if multiple elements at once should be specified a Layout should be used.